chart/environment-values/values.openwifi-qa.disable-tls.yaml

```diff
@@ -0,0 +1,29 @@
+# NOTE: using of this values files will require you to use http schema in 'openwifi.system.uri.private' (and maybe 'openwifi.system.uri.public') configuration properties (i.e. http://owgw-owgw:17002)
+owgw:
+  configProperties:
+    openwifi.security.restapi.disable: "true"
+
+owsec:
+  configProperties:
+    openwifi.security.restapi.disable: "true"
+
+owfms:
+  configProperties:
+    openwifi.security.restapi.disable: "true"
+
+owprov:
+  configProperties:
+    openwifi.security.restapi.disable: "true"
+
+owanalytics:
+  configProperties:
+    openwifi.security.restapi.disable: "true"
+
+owsub:
+  configProperties:
+    openwifi.security.restapi.disable: "true"
+
+haproxy:
+  service:
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
```

Comment:
Do you know if the fileupload and rttys component of the gateway service will also have a plaintext listener if `openwifi.security.restapi.disable` is set to true? If not I think this may lead to problems.

Reply:
Good catch!
It seems that fileuploader is still using TLS even with this option:
```
# REST API
bash-5.1# curl http://localhost:16002 && echo
{"ErrorCode":400,"ErrorDescription":"Unsupported HTTP Method","ErrorDetails":"GET"}
# Rttys viewport
bash-5.1# curl http://localhost:5913/rtty/connect  -I
HTTP/1.1 200 OK
Date: Mon, 02 May 2022 11:36:18 GMT
Connection: Keep-Alive
Transfer-Encoding: chunked
Access-Control-Allow-Origin: *
Access-Control-Allow-Headers: *
Access-Control-Max-Age: 86400
Access-Control-Allow-Methods: GET, OPTIONS, HEAD
Keep-Alive: timeout=120
Accept-Ranges: bytes

# fileuploader http
bash-5.1# curl http://localhost:16003/v1/upload/qwe -I
curl: (56) Recv failure: Connection reset by peer
# fileuploader https with TLS check skip
bash-5.1# curl https://localhost:16003/v1/upload/qwe -Ik
HTTP/1.1 501 Not Implemented
Connection: Close
```

I'll let the Stephane know that this is an issue